src/lib/migration-chunks/validation/editor-layout.ts

```diff
@@ -69,6 +77,59 @@ class NonExistingDeletes implements EditorLayoutValidation {
   }
 }
 
+class InvalidFieldMove implements EditorLayoutValidation {
+  validate (intent: Intent, context: ValidationContext): string | string[] {
+    if (intent.getRawType() !== 'contentType/moveFieldInEditorLayout') {
+      return
+    }
+
+    const moveIntent = (intent as EditorLayoutMoveFieldIntent)
+    if (!moveIntent.getFieldId()) {
+      return editorLayoutErrors.moveField.MISSING_FIELD_ID()
+    }
+
+    const fieldExists = context.fields.contentTypeFields[moveIntent.getContentTypeId()].has(moveIntent.getFieldId()) &&
```

Comment:
What about destructuring the context and errors a bit? It's currently hard to read.